app/bundles/InstallBundle/InstallFixtures/ORM/LeadFieldData.php

```diff
@@ -42,7 +42,7 @@ public function setContainer(ContainerInterface $container = null)
      */
     public function load(ObjectManager $manager)
     {
-        $fields       = FieldModel::$coreFields;
+        $fields       = array_merge(FieldModel::$coreFields, FieldModel::$coreFields);
```

Comment:
oh no, but this was changed in another branch, which is why I suggested to better just PR one for these features.